code/integration-tests/runtime-tests/tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    // this is specific with augmented overrides
+    "paths": {
+      // this is the package name we use (in the interface imports, --package for generators)
+      "sample-polkadotjs-typegen/*": ["src/*"],
+      // here we replace the @polkadot/api augmentation with our own, generated from chain
+      "@polkadot/api/augment": ["src/interfaces/augment-api.ts"],
+      // replace the augmented types with our own, as generated from definitions
+      "@polkadot/types/augment": ["src/interfaces/augment-types.ts"]
+    },
+    "lib": [
+      "ES2021.String"
+    ],
+    // some other options, whatever you want for your environment
+    "target": "es2017",
+    "module": "commonjs",
+    "jsx": "preserve",
+    "declaration": true,
+    "strict": true,
+    "noImplicitAny": true,
+    "noUnusedLocals": true,
+    "noImplicitReturns": true,
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "baseUrl": ".",
+    "skipLibCheck": true,
```

Comment:
why?

Reply:
For some reason when this is enabled, it was also compiling types on node modules even though it is added on excluded folder. As the fastest solution, just added this line here.